src/protocolgame.cpp

```diff
@@ -1334,13 +1365,111 @@ void ProtocolGame::sendBasicData()
 		msg.add<uint32_t>(0);
 	}
 	msg.addByte(player->getVocation()->getClientId());
+	msg.addByte(1);
```

Comment:
elaborate? I guess its the byte for enabling prey, I dont think prey should be always enabled.

Reply:
1 prey is enabled by default, 2 for premium and 3 purchasable

Reply:
no, its not, while you are no vocation, you arent able to use prey.

Reply:
true, I forgot that

Reply:
Maybe this need a check to avoid player with no vocation using prey.